conjure-java-server-verifier/src/test/java/com/palantir/conjure/java/verification/server/undertest/UndertowServerUnderTestRule.java

```diff
@@ -40,7 +41,7 @@ protected void before() throws Throwable {
         Service endpoints = AutoDeserializeServiceEndpoints.of(service);
 
         ConjureHandler handler = new ConjureHandler();
-        ServiceContext context = ServiceContext.builder()
+        ServiceContext context = ConjureContext.builder()
                 .serializerRegistry(ConjureSerializerRegistry.getDefault())
```

Comment:
There's no reason to expose the serializer registry, serializers should be added directly to the `ConjureContext`. By default, we should use the `ConjureSerializerRegistry.getDefault()` configuration.